tests/test-cases/Group6-VIC-Machine/6-15-Syslog.robot

```diff
@@ -32,7 +32,6 @@ Get Remote PID
     Should Not Be Empty  ${pid}
     [Return]  ${pid}
 
-*** Test Cases ***
 Verify VCH remote syslog
```

Comment:
Please make sure container logs are forwarded to the syslog server too. If the test misses, please help to add.

Reply:
Yes. Container log is checked by this test case